config/locales/server.en.yml

```diff
@@ -2432,6 +2432,7 @@ en:
       search_tokenize_chinese_enabled: "You must disable 'search_tokenize_chinese' before enabling this setting."
       search_tokenize_japanese_enabled: "You must disable 'search_tokenize_japanese' before enabling this setting."
       discourse_connect_cannot_be_enabled_if_second_factor_enforced: "You cannot enable DiscourseConnect if 2FA is enforced."
+      delete_rejected_email_after_days: "This setting cannot be set lower than the delete_email_logs_after_days setting or greater than 36500."
```

Comment:
We should avoid hardcoding `36500` into the translation string and pass in the value to the translation. This will help to avoid a case where the `MAX` in our code is updated but not the value in the translation string.